tests/certification/bindings/azure/eventhubs/eventhubs_test.go

```diff
@@ -0,0 +1,181 @@
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+	"github.com/dapr/components-contrib/tests/certification/flow/simulate"
+	"github.com/dapr/components-contrib/tests/certification/flow/watcher"
+	dapr "github.com/dapr/go-sdk/client"
+	"github.com/dapr/go-sdk/service/common"
+)
+
+const (
+	numMessages = 100
+)
+
+func TestEventhubBinding(t *testing.T) {
+	logger := logger.NewLogger("dapr.components")
+	out_component := bindings_loader.NewOutput("azure.eventhubs", func() bindings.OutputBinding {
+		return eventhubs.NewAzureEventHubs(logger)
+	})
+	in_component := bindings_loader.NewInput("azure.eventhubs", func() bindings.InputBinding {
+		return eventhubs.NewAzureEventHubs(logger)
+	})
+	secrets_components := secretstores_loader.New("local.env", func() secretstores.SecretStore {
+		return secretstore_env.NewEnvSecretStore(logger)
+	})
+
+	ports, _ := dapr_testing.GetFreePorts(3)
+	grpcPort := ports[0]
+	httpPort := ports[1]
+	appPort := ports[2]
+
+	consumerGroup1 := watcher.NewUnordered()
+	consumerGroup2 := watcher.NewUnordered()
+
+	sendAndReceive := func(ctx flow.Context) error {
+
+		client, err := dapr.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "dapr init failed")
+
+		// Define what is expected
+		outputmsg := make([]string, numMessages)
+		for i := 0; i < numMessages; i++ {
+			outputmsg[i] = fmt.Sprintf("output binding: Message %03d", i)
+		}
+		consumerGroup1.ExpectStrings(outputmsg...)
+		time.Sleep(20 * time.Second)
+
+		// Send events from output binding
+		for _, msg := range outputmsg {
+			ctx.Logf("Sending eventhub message: %q", msg)
+
+			err := client.InvokeOutputBinding(
+				ctx, &dapr.InvokeBindingRequest{
+					Name:      "azure-eventhubs-binding",
+					Operation: "create",
+					Data:      []byte(msg),
+				})
+			require.NoError(ctx, err, "error publishing message")
+		}
+
+		// Assert the observed messages
+		consumerGroup1.Assert(ctx, time.Minute)
+		return nil
+	}
+
+	// Application logic that tracks messages from eventhub.
+	application := func(ctx flow.Context, s common.Service) (err error) {
+		// Simulate periodic errors.
+		sim := simulate.PeriodicError(ctx, 100)
+		// Setup the binding endpoints
+		err = multierr.Combine(err,
+			s.AddBindingInvocationHandler("azure-eventhubs-binding", func(_ context.Context, in *common.BindingEvent) ([]byte, error) {
+				consumerGroup1.Observe(string(in.Data))
+				if err := sim(); err != nil {
+					return nil, err
+				}
+				ctx.Logf("Receiving eventhubs message: %s", string(in.Data))
+				return []byte("{}"), nil
+			}))
+		return err
+	}
+
+	publishMessages := func(ctx flow.Context) error {
+		// Define what is expected
+		outputmsg := make([]string, numMessages)
+		for i := 0; i < numMessages; i++ {
+			outputmsg[i] = fmt.Sprintf("publish messages to device: Message %03d", i)
+		}
+		consumerGroup2.ExpectStrings(outputmsg...)
+		cmd := exec.Command("/bin/bash", "../../../tests/scripts/send-iot-device-events.sh")
```

Comment:
Please do not do this. Move the script into the same folder as the certification test (with `git mv`) and then change the location.